www_src/components/basic-element/basic-element.jsx

```diff
@@ -40,11 +40,11 @@ var El = React.createClass({
       zIndex: this.props.zIndex,
       touchactive: false
     };
+    this.buttonStyle = initial;
```

Comment:
Would have to check the source code, but using `getInitialState` to reset element state is a common pattern/has no downsides AFAIK. This is fine, although it does open a chance of potential bugs happening from inadvertently modifying the reference to `this.buttonStyle`